### PR TITLE
AAE-11466: allow showing maven downloads on build-and-tag action

### PR DESCRIPTION
**Pull request by atchertchian**
The new `verbose` input should allow us to debug issues related to maven downloads.

### CODE REVIEW

**Comment by atchertchian** [comment]
Will rework this PR so that we still have a way to avoid showing downloads by default, but can easily show them

File: .github/actions/maven-build-and-tag/action.yml [hunk 1]
```diff
@@ -60,13 +64,26 @@ runs:
...
         distribution: ${{ inputs.java-distribution }}
 
+    - name: Compute maven options
+      id: compute-maven-options
+      shell: bash
+      env:
+        INPUT_VERBOSE: ${{ inputs.verbose }}
+      run: |
+        NTP=''
+        if [[ '$INPUT_VERBOSE' == 'true' ]]
```
Comment (gionn) [inline .github/actions/maven-build-and-tag/action.yml:74]: double quotes or won't expand $INPUT_VERBOSE

Comment (atchertchian) [reply .github/actions/maven-build-and-tag/action.yml]: Fixed, thanks

[the remark this answers is not in the document]
Comment (atchertchian) [reply .github/actions/maven-build-and-tag/action.yml]: This is still buggy: the evaluation is `true` without quotes...

**Review by gionn — APPROVED** [review]

**Pull request merged**